roboteam_ai/src/ApplicationManager.cpp

```diff
@@ -103,7 +103,7 @@ void ApplicationManager::notifyTreeStatus(bt::Node::Status status) {
     case Status::Running:break;
     case Status::Success:ROS_INFO_STREAM("Status returned: Success");
         ROS_INFO_STREAM(" === TREE CHANGE === ");
-        BTFactory::setCurrentTree("haltStrategy");
+        BTFactory::setCurrentTree("keeperStrategy");
```

Comment:
you need to initialize on halt

Reply:
This is not initialization, but the tree it switches to after completing a tree.